src/context.rs

```diff
@@ -385,6 +481,29 @@ impl BlockContext {
         ((left * 2 + above) + bsl) as usize * PARTITION_PLOFFSET
     }
 
+    pub fn update_partition_context(&mut self, bo: &BlockOffset,
+                                subsize : BlockSize, bsize: BlockSize) {
+#[allow(dead_code)]
```

Comment:
Is this still needed? I think because you commented out bw and bh you don't need it.

Reply:
Yes, we need it. It is one of necessary function to update partition contexts.
Before EXT_PARTITION_TYPES is enabled, very strangely libaom copies width size (in mi)  of
to both up a

Reply:
There is a patch for libaom, which I am trying to learn how to merge to rav1e or other separate libaom for rav1e.
However, the rav1e works fine w/o for now since the partition is not enabled yet thus lookup tables, esp. partition context, are not used.